credentials/generate-revocation-set.py

```diff
@@ -491,13 +591,12 @@ def main(use_main_net_dcld: str, use_test_net_dcld: str, use_main_net_http: bool
         # 10. Iterate through the Revoked Certificates List
         for revoked_cert in crl_file:
             try:
-                revoked_cert_issuer = revoked_cert.extensions.get_extension_for_oid(
-                    x509.CRLEntryExtensionOID.CERTIFICATE_ISSUER).value.get_values_for_type(x509.DirectoryName).value
-
+                revoked_cert_issuer = base64.b64encode(revoked_cert.extensions.get_extension_for_oid(
+                    x509.CRLEntryExtensionOID.CERTIFICATE_ISSUER).value.get_values_for_type(x509.DirectoryName)[0].public_bytes()).decode("utf-8")
                 if revoked_cert_issuer is not None:
                     # check if this really are the same thing
                     if revoked_cert_issuer != certificate_authority_name_b64:
-                        logging.warning("CRL Issuer is not CRL File Issuer, continue...")
+                        logging.debug("CRL Issuer is not CRL File Issuer, continue...")
```

Comment:
Is there a reason why this one is "debug" while all the others where "..., continue"?

Reply:
I've fixed this discrepancy.